### PR TITLE
Theming: standard button semantic slots, fix disabled colors

### PR DESCRIPTION
**Pull request by phkuo**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Add semantic slots for the standard button, plus fix the name of an incorrectly named semantic slot. Also fix disabled colors in various places.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts [hunk 1]
```diff
@@ -29,15 +29,19 @@ const iconStyle = {
...
 
   return {
     root: [
       getFocusStyle(theme, -1),
       theme.fonts.medium,
       {
-        // this transparent border converts to the correct colors in HC mode
         boxSizing: 'border-box',
-        border: '1px solid transparent',
+        border: '1px solid ' + border,
```
Comment (joschect) [inline packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts]: Can border ever be undefined or null? If so you'll need to have a check for it as javascript will concat it to '1px solidundefined'

Comment (phkuo) [reply packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts]: nope, our theming module insures it's always defined

**Comment by phkuo** [comment]
@dzearing I standardized disabled text color in buttons from neutralTertiary to neutralTertiaryAlt, which is the color we use for disabled in other controls, hence the screener failures.

**Comment by betrue-final-final** [comment]
The screener seems to be doing its job.

**Comment by phkuo** [comment]
Why do I have to review my own PR?!
@micahgodbolt @dzearing 
image

**Comment by betrue-final-final** [comment]
I don't know. We changed the change list comment

________________________________

**Comment by phkuo** [comment]
Yeesh, thanks @battletoilet !

**Pull request merged**